tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/transform/C2jModelToGeneratorModelTransformer.java

```diff
@@ -643,12 +906,48 @@ Operation convertOperation(C2jOperation c2jOperation) {
                     }
                 }
             }
+            Map<String, Map<String, String>> operationContextParams = c2jOperation.getOperationContextParams();
+            if (operationContextParams != null )
+            {
+                
+                String path = null;
+                //get path from context param
+                for (Map.Entry<String, Map<String, String>> outerEntry: operationContextParams.entrySet()) {
+                    String outerKey = outerEntry.getKey();
+                    Map<String, String> innerMap = outerEntry.getValue();
+                    String pathValue = innerMap.get("path");
+                    if(pathValue != null)
+                    {
+               
+                        OperationContextCppCodeGenerator ctxt = new OperationContextCppCodeGenerator();
+
+                        String value = JmespathExpression.parse(pathValue)
```

Comment:
what does this return and why are we ignoring it?

Reply:
The output is captured in the context which we use. The return is simple a par of expression and shape which we use internally in tree traversal. I can remove the "String value = "